src/PhpGenerator/ClassType.php

```diff
@@ -99,7 +99,7 @@ public static function from($from)
 	}
 
 
-	public function __construct($name = NULL, PhpNamespace $namespace = NULL)
+	public function __construct($name, PhpNamespace $namespace = NULL)
```

Comment:
This is BC break when you use `new ClassType)->setName(...)` (used i.e by nette/di).